src/InteractiveWindow/Editor/InteractiveWindow.csproj

```diff
@@ -124,6 +124,16 @@
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
     <PublicAPI Include="PublicAPI.Shipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Dependencies\VisualStudioEditor\VisualStudioEditor.csproj">
```

Comment:
If we decide my other comment about merging these is daft, shouldn't VisualStudioEditor depend on VisualStudioText? Can you actually consume one without the other?